install.sh

```diff
@@ -3947,18 +3994,24 @@ install_into_existing_repositories() {
 
     git config --global githooks.previous.searchdir "$RAW_START_DIR"
 
-    LOCAL_REPOSITORY_LIST=$(find "$START_DIR" -type d -name .git 2>/dev/null)
+    REPOSITORY_LIST=$(
+        find "$START_DIR" \( -type d -and -name .git \) -or \
+            \( -type f -and -name HEAD -and -not -path "*/.git/*" \) 2>/dev/null
+    )
 
     # Sort the list if we can
     if sort --help >/dev/null 2>&1; then
-        LOCAL_REPOSITORY_LIST=$(echo "$LOCAL_REPOSITORY_LIST" | sort)
+        REPOSITORY_LIST=$(echo "$REPOSITORY_LIST" | sort)
     fi
 
     IFS="$IFS_NEWLINE"
-    for EXISTING in $LOCAL_REPOSITORY_LIST; do
+    for EXISTING in $REPOSITORY_LIST; do
         unset IFS
 
         EXISTING_REPO_ROOT=$(dirname "$EXISTING")
+        if ! is_git_repo "${EXISTING_REPO_ROOT}"; then
```

Comment:
This is running on the `dirname` of the repo path, how is that going to work with absolute paths?
Shouldn't this be `if ! is_git_repo "${EXISTING}"; then` ?

Reply:
Please check this I think this is a bug. /final